stage1/init/common/pod.go

```diff
@@ -372,6 +368,18 @@ func findBinPath(p *stage1commontypes.Pod, appName types.ACName, app types.App,
 	return binPath, nil
 }
 
+// shiftFiles shifts filesToshift by the amounts specified in uidRange
```

Comment:
https://twitter.com/baronboulle/status/586278125199351808

On 11 August 2016 at 17:42, Krzesimir Nowak <notifications@github.com>
wrote:

> In stage1/init/common/pod.go
> <https://github.com/coreos/rkt/pull/3058#discussion_r74447166>:
>
> > @@ -372,6 +368,18 @@ func findBinPath(p *stage1commontypes.Pod, appName types.ACName, app types.App,
> >  	return binPath, nil
> >  }
> >
> > +// shiftFiles shifts filesToshift by the amounts specified in uidRange
>
> Not enough shift.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coreos/rkt/pull/3058/files/9af924ee0d930eced4806010443e02b920edf158#r74447166>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACewN_rAGK8iVgwNHZfTxd8ZPhwi1FqMks5qe0LVgaJpZM4JiPEk>
> .
>